app/src/main/java/solendsdkfacade/SolendSdkFacade.kt

```diff
@@ -0,0 +1,104 @@
+package solendsdkfacade
+
+import P2pSdk
+import com.google.gson.Gson
+import org.p2p.solanaj.rpc.NetworkEnvironment
+import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
+import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManager
+import org.p2p.wallet.utils.Base58String
+import solendsdkfacade.mapper.SolendMethodResultMapper
+import solendsdkfacade.model.SolendEnvironment
+import solendsdkfacade.model.SolendFeePayerTokenData
+import solendsdkfacade.model.SolendMethodResultSuccess
+import solendsdkfacade.model.SolendMethodResultSuccess.SolendDepositTransactions
+import solendsdkfacade.model.SolendMethodResultSuccess.SolendWithdrawTransactions
+import timber.log.Timber
+import kotlinx.coroutines.withContext
+
+private const val TAG = "P2PSdkFacade"
+
+class SolendSdkFacade(
+    private val p2pSdk: P2pSdk,
```

Comment:
Strange naming.
We are moving from P2P, so let's make some universal name. 
SolendSdk or smth like that

Reply:
I don't have any choice now, we need to use naming provided from JNI Rust interface. Renaming in process.